app/routes.py

```diff
@@ -1,2 +1,238 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+import datetime
+import requests
+import os
+from dotenv import load_dotenv
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods = ["GET"])
+def task_index():
+    sort_query = request.args.get("sort")
+    if sort_query == "asc":
+        tasks = Task.query.order_by(Task.title.asc())
+    elif sort_query == "desc":
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+        
+    tasks_response = []
+    for task in tasks:
+        tasks_response.append(task.to_json())
+    return jsonify(tasks_response), 200
+
+
+@tasks_bp.route("", methods = ["POST"])
+def tasks():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+        return make_response({ "details": "Invalid data"}, 400)
+    else:  
+        new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
+        db.session.add(new_task)
+        db.session.commit()
+        return make_response(jsonify({"task": new_task.to_json()}), 201)
+
+@tasks_bp.route("/<task_id>", methods = ["GET", "PUT", "DELETE"])
+def handle_tasks(task_id):
+    task = Task.query.get(task_id)
+    if task is None:
+        return make_response("", 404)
```

Comment:
Great job Priscille! Flask has a really cool method `get_or_404()` that you can use to refine these next few lines into: 
```suggestion
    task = Task.query.get_or_404(task_id)

```